website_and_docs/content/project/governance/_index.html

```diff
@@ -60,30 +83,30 @@ <h3 id="users">Users</h3>
       <h3 id="contributors">Contributors</h3>
 
       Contributors are community members who contribute in concrete ways to the project,
-      most often in the form of code and/or documentation. Anyone can become a Contributor,
-      and contributions can take many forms, e.g.:
+      most often in the form of code and/or documentation in the respective project repository. 
+      Anyone can become a contributor, and contributions can take many forms, other than code contributions, e.g.:
       <ul>
           <li>
             <p class="m-0">
-              Help other users through any of the <a href="#communication-channels">communication channels</a>
+              Actively help other users through any of the <a href="#communication-channels">communication channels</a>
               made for that purpose.
             </p>
           </li>
           <li>
               Triage GitHub issues.
           </li>
-          <li>
-              Organize Selenium meetups.
+           <li>
+              Translate Selenium Documentation in languages project support. 
           </li>
-          <li>
-              Organize and collaborate in Selenium Conferences.
+         <li>
+              Organize and collaborate in Selenium meetups, events and conferences.
           </li>
       </ul>
 
       <p>
         There is no expectation of commitment to the project, no specific skill requirements,
         and no selection process.
-        Some Contributors might have some basic privileges to the GitHub repos, based on
+        Some Contributors might have some basic privileges to the GitHub repos to which they have contributed, based on
```

Comment:
If they are contributors, it is because they have contributed. This phrase is redundant.